### ISSUE TITLE
Make the query accessible from the output filter

### ISSUE THREAD

**Issue by billerickson** [billerickson/display-posts-shortcode#198]
The following filters include the `$listing` query:
 - `display_posts_shortcode_post_class`
- `display_posts_shortcode_wrapper_open`
- `display_posts_shortcode_wrapper_close`

But the output filter does not. 

Rather than adding yet another parameter to that filter, I'll probably rename `$listing` to `$dps_listing` and make it globally accessible, like the global `$post` variable.

**Comment by airdrummer** [comment]
so $dps_listing->current_post contains the index(count) of the current post?

**Comment by billerickson** [comment]
Yes it does. Here's a [list of all the properties accessible](https://codex.wordpress.org/Class_Reference/WP_Query#Properties) from `$dps_listing`.
